FirebaseAdmin/FirebaseAdmin/AppOptions.cs

```diff
@@ -58,5 +60,10 @@ internal AppOptions(AppOptions options)
         /// JSON.</para>
         /// </summary>
         public string ServiceAccountId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the HttpClientFactory to use when making Firebase requirests.
+        /// </summary>
+        public HttpClientFactory ClientFactory { get; set; } = new HttpClientFactory();
```

Comment:
Lets call it `HttpClientFactory` without shortening the name.

Reply:
I'm happy to change, though I did notice that FirebaseUserManagerArgs uses "ClientFactory" as well.  In the end, I'm indifferent, but would recommend sticking with one naming convention or the other.

Reply:
We should be consistent with other Google libraries like Google API client, at least in our public API surface: https://github.com/googleapis/google-api-dotnet-client/blob/master/Src/Support/Google.Apis/Services/BaseClientService.cs#L68

+1 to renaming the internal fields too.